blocks/intro/__title/_page/intro__title_page_catalog.css

```diff
@@ -0,0 +1,14 @@
+.intro__title_page_catalog{
+  max-width: 357px;
+}
+@media (max-width: 1199px){
+  .intro__title_page_catalog {
+    padding: 177px 0 130px;
+  }
+}
+@media (max-width: 659px){
+  .intro__title_page_catalog {
+    max-width: 172px;
```

Comment:
@MarPostovik While we've used `max-width` primarily for line breaking on larger screens to match the design, our priority on mobile devices is ensuring text _elements aren't too close to the device edges_. Setting the max width to `280px` for this element will be just enough to guarantee 20px space on both the right and left sides, accommodating even the smallest screens we need to consider, such as 320px wide. 

I recommend limiting the width of this title to `280px`, similar to the `.intro__title_page_form element`. Note that the visuals of this section at 320px won't change 😀

Reply:
done ✅